src/commcare_cloud/ansible/roles/commcarehq/templates/supervisor_formplayer_spring.conf.j2

```diff
@@ -1,7 +1,7 @@
 [program:{{ project }}-{{ deploy_env }}-formsplayer-spring]
 environment={% for name, value in item.env_vars.items() %}{% if value %}{{ name }}="{{ value }}",{% endif %}{% endfor %}
 
-command=java {% if app_processes_config.newrelic_javaagent %}-javaagent:/home/cchq/www/{{ deploy_env }}/newrelic/newrelic.jar{% endif %} -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.port=9998 -Xmx{{ app_processes_config.formplayer_memory }} -XX:MaxMetaspaceSize=128m -XX:-OmitStackTraceInFastThrow -Xss1024k -XX:NativeMemoryTracking=summary -XX:+UnlockDiagnosticVMOptions -jar {{ www_home }}/formplayer_build/current/formplayer.jar
+command=java {% if app_processes_config.newrelic_javaagent %}-javaagent:/home/cchq/www/{{ deploy_env }}/newrelic/newrelic.jar{% endif %} -Xmx{{ app_processes_config.formplayer_memory }} -XX:MaxMetaspaceSize=128m -XX:-OmitStackTraceInFastThrow -Xss1024k -jar {{ www_home }}/formplayer_build/current/formplayer.jar
```

Comment:
Noting (for myself) the settings removed:

```
    -Dcom.sun.management.jmxremote.authenticate=false
    -Dcom.sun.management.jmxremote.ssl=false
    -Dcom.sun.management.jmxremote=true
    -Dcom.sun.management.jmxremote.port=9998
    -XX:NativeMemoryTracking=summary
    -XX:+UnlockDiagnosticVMOptions
```

Reply:
In case it's helpful for future work: the top 4 allow someone to remotely connect VisualVM to the process by sshing into the machine with a tunnel on port 9998